brightlayer-ui/react-component-library
# Remove mui styles package

Fixes BLUI-3659 && #633.


#### Changes proposed in this Pull Request:
- Remove mui styles package


#### To Test:
- `yarn start:reactdev` and make sure everything is working as expected. 

>Note: dev docs is currently using the latest icons and themes alpha packages.

Unit tests are not going to pass on CI for now
